cdb2api/cdb2api.c

```diff
@@ -927,8 +930,10 @@ void cdb2_set_comdb2db_config(const char *cfg_file)
                 cfg_file);
     memset(CDB2DBCONFIG_NOBBENV, 0, sizeof(CDB2DBCONFIG_NOBBENV) /* 512 */);
     if (cfg_file != NULL) {
+        cdb2cfg_override = 1;
         strncpy(CDB2DBCONFIG_NOBBENV, cfg_file, 511);
     } else {
+        cdb2cfg_override = CDB2CFG_OVERRIDE_DEFAULT;
```

Comment:
It seems like this line should be moved into the reset_the_configuration() function?